db/memtable_list_test.cc

```diff
@@ -258,7 +260,7 @@ TEST_F(MemTableListTest, GetTest) {
   ASSERT_EQ(1, mem->num_deletes());
 
   // Add memtable to list
-  list.Add(mem, &to_delete);
+  list.Add(mem, &to_delete, 0);
```

Comment:
What does the third argument mean here?

Reply:
the amount of memory currently being used by mutable memtable. This number together with memtable list size are checked against `max_write_buffer_size_to_maintain_` in `TrimHistory` to decide whether to drop memtable list

Reply:
Yeah, given this, can we change the third argument to `mem->ApproximateMemoryUsage()` (which is likely close to 0)?